README.md

```diff
@@ -1,28 +1,27 @@
-README for bdii package
-=======================
+# README for bdii package
 
-Documentation: http://bdii.readthedocs.io
+Documentation: [bdii.readthedocs.io](http://bdii.readthedocs.io)
 
 ## Function
 
 The Berkeley Database Information Index (BDII) consists of two or more
 standard LDAP databases that are populated by an update process.
 Port forwarding is used to enable one or more databases to serve data
-while one database is being refreshed.  The databases are refreshed
-cyclically.  Any incoming connection is forwarded to the most recently
+while one database is being refreshed. The databases are refreshed
+cyclically. Any incoming connection is forwarded to the most recently
 updated database, while old connections are allowed to linger until
 it is the turn of their database to be refreshed and restarted.
 The update process obtains LDIF from either doing an ldapsearch on LDAP
 URLs or by running a local script (given by a URL with "file" protocol)
-that generates LDIF.  The LDIF is then inserted into the LDAP database.
+that generates LDIF. The LDIF is then inserted into the LDAP database.
 Options exist to update the list of LDAP URLs from a web page and
 to use an LDIF file from a web page to modify the data before it is
 inserted into the database.
 
 ## Cache use
 
 Whenever a remote server is contacted and the ldapsearch command times out
-the update process tries to find an (old) cached entry in the var/cache/ 
+the update process tries to find an (old) cached entry in the `/var/cache/`
 directory. If no entry is found a message is printed to the logfile.
 
 ! Attention !
```

Comment:
What's with the wierd punctuation? We should perhaps make it **bold**?, _i.e._ 
! Attention ! --> **Attention!*